.ci/int-test-helper/run-tests

```diff
@@ -22,7 +22,7 @@ SOURCE_PATH="$(pwd)"
 echo "Run integration tests in source path ${SOURCE_PATH}"
 
 echo "Run integration tests in source path ${SOURCE_PATH}"
-ginkgo -timeout=60m -v -progress -no-color -seed=17 -fail-fast ./test/integration  -- \
+ginkgo -timeout=60m -v -progress -noColor -seed=17 -failFast ./test/integration  -- \
```

Comment:
These are the old settings of ginkgo version 1. With version 2 they have been changed. You need to install the new ginkgo version locally. Or have you had problems with the tests running on concours.

Reply:
I changed it back.